src/transformers/image_processing_utils.py

```diff
@@ -662,7 +662,13 @@ def center_crop(
         )
 
 
-VALID_SIZE_DICT_KEYS = ({"height", "width"}, {"shortest_edge"}, {"shortest_edge", "longest_edge"}, {"longest_edge"})
+VALID_SIZE_DICT_KEYS = (
+    {"height", "width"},
+    {"shortest_edge"},
+    {"shortest_edge", "longest_edge"},
+    {"longest_edge"},
+    {"max_height", "max_width"},
```

Comment:
If the models are now using `pad_size` - do we need the `max_size` arguments here?